rfcs/text/0008-threat-intel.md

```diff
@@ -23,30 +23,38 @@ Stage 1: Describe at a high level how this change affects fields. Which fieldset
 
 ### Proposed New Fields for Threat fieldset
 
-  * threat.time_first_seen
+  * threat.ioc.time_first_seen
     _The date and time when intelligence souce first reported sighting this indicator._
-  * threat.time_last_seen
+  * threat.ioc.time_last_seen
     _The date and time when intelligence source last reported sighting this indicator._
-  * threat.sightings
+  * threat.ioc.sightings
     _Number of times this indicator was observed conducting threat activity._
   * threat.type
     _Type of indicator as reprsented by Cyber Observable in STIX 2.0_
-  * threat.description
+  * threat.ioc.description
     _Describes the type of action conducted by the threat._
-  * threat.tlp
+  * threat.ioc.tlp
     _Traffic Light Protocol, which dictates sharing policies_
-  * threat.classification
+  * threat.ioc.classification
```

Comment:
```suggestion
threat.malware.{type,family,name}
```

Reply:
I agree with splitting `classification` into multiple fields. This makes room for more fine grained classification 👍 

However I think @SHolzhauer makes a good point. If the `.malware.` nesting constrains the usage of these fields, perhaps we could find a more general name? Or have the classification fields directly under `threat.ioc.`?

If @peasead's intention was to define those as "malware-specific" classification, perhaps we can add another field like `threat.ioc.kind: malware`. So this would make it possible to use the classification fields for other types of IOCs as well.

Reply:
Thanks @webmat and @SHolzhauer

The thought around moving malware up from underneath `threat.ioc.` was because the malware family, name, and type aren't IOCs, they're _descriptors_ of the IOC. 
```
"threat": {
    "ioc": {
      "file": {
        "hash": {
          "sha1": "bfb7759a67daeb65410490b4d98bb9da7d1ea2ce"
        }
      }
    },
    "malware": {
      "name": "CryptoMinerX.B",
      "family": "CryptoMinerX",
      "type": "cryptominer"
    },
```

@webmat so are you saying that if we added `threat.ioc.kind: malware` (which I like), then we'd re-nest malware under ioc?

Reply:
I like this `threat.ioc.{type,family,name}`.

Let me throw some `.kind`  ` ```suggestion` into the RFC.

Reply:
`.type` has different context for analysts when nested under `threat.ioc.*` vs `threat.malware.*` . `threat.ioc` will be interpreted by analysts as file name, IPV4,IPV6, file hash etc. `malware` context becomes more useful when its particular variant (name), family and objective(type) are available. I would recommend using [MAEC 5.0](https://maecproject.github.io/releases/5.0/MAEC_Core_Specification.pdf) standard as guidance for attributes to use for enumerating malware families.

Reply:
This looks to be addressed here in `threat.yml`.
https://github.com/elastic/ecs/blob/1347f2ba4c0ef6c00d5ffccee7fa2ad854f631d0/rfcs/text/0008/threat.yml#L31